src/people.cpp

```diff
@@ -110,11 +140,26 @@ void Guests::Load(Loader &ldr)
 		case 0:
 			break;
 		case 1:
+		case 2:
 			this->start_voxel.x = ldr.GetWord();
 			this->start_voxel.y = ldr.GetWord();
 			this->daily_frac = ldr.GetWord();
 			this->next_daily_index = ldr.GetWord();
 			this->free_idx = ldr.GetLong();
+
+			if (version > 1) {
+				this->complaint_counter_hunger       = ldr.GetWord();
+				this->complaint_counter_thirst       = ldr.GetWord();
+				this->complaint_counter_waste        = ldr.GetWord();
+				this->complaint_counter_litter       = ldr.GetWord();
+				this->complaint_counter_vandalism    = ldr.GetWord();
+				this->time_since_complaint_hunger    = ldr.GetLong();
+				this->time_since_complaint_thirst    = ldr.GetLong();
+				this->time_since_complaint_waste     = ldr.GetLong();
+				this->time_since_complaint_litter    = ldr.GetLong();
+				this->time_since_complaint_vandalism = ldr.GetLong();
+			}
```

Comment:
This won't get better as you do need to write out all data one way or another, but that's ok.